### DISCUSSION TITLE
Configuring Sveltekit + Storybook 7 CSF

### DISCUSSION THREAD

**Discussion by ctwheels** [storybookjs/storybook#20775] [Help]
I have a very simple button component for testing. Here's my config.

When I run the `storybook` command, I get the following errors, any idea how I can fix this?

> storybook /workspace/web
> storybook dev -p 6006
> 
> @storybook/cli v7.0.0-beta.13
> 
> info => Starting manager..
> WARN 🚨 Extraction error on src/components/Button/Button.stories.svelte: Error: No matching story indexer found for /workspace/web/src/components/Button/Button.stories.svelte
> /workspace/web/node_modules/.pnpm/@storybook+core-server@7.0.0-beta.13/node_modules/@storybook/core-server/dist/cjs/utils/StoryIndexGenerator.js:178
>         throw new Error(`No matching story indexer found for ${absolutePath}`);
>               ^
> 
> Error: No matching story indexer found for /workspace/web/src/components/Button/Button.stories.svelte
>     at StoryIndexGenerator.extractStories (/workspace/web/node_modules/.pnpm/@storybook+core-server@7.0.0-beta.13/node_modules/@storybook/core-server/dist/cjs/utils/StoryIndexGenerator.js:178:15)
>     at /workspace/web/node_modules/.pnpm/@storybook+core-server@7.0.0-beta.13/node_modules/@storybook/core-server/dist/cjs/utils/StoryIndexGenerator.js:121:62
>     at /workspace/web/node_modules/.pnpm/@storybook+core-server@7.0.0-beta.13/node_modules/@storybook/core-server/dist/cjs/utils/StoryIndexGenerator.js:107:37
>     at Array.map (<anonymous>)
>     at /workspace/web/node_modules/.pnpm/@storybook+core-server@7.0.0-beta.13/node_modules/@storybook/core-server/dist/cjs/utils/StoryIndexGenerator.js:105:45
>     at Array.map (<anonymous>)
>     at StoryIndexGenerator.updateExtracted (/workspace/web/node_modules/.pnpm/@storybook+core-server@7.0.0-beta.13/node_modules/@storybook/core-server/dist/cjs/utils/StoryIndexGenerator.js:103:39)
>     at StoryIndexGenerator.ensureExtracted (/workspace/web/node_modules/.pnpm/@storybook+core-server@7.0.0-beta.13/node_modules/@storybook/core-server/dist/cjs/utils/StoryIndexGenerator.js:120:16)
>     at StoryIndexGenerator.initialize (/workspace/web/node_modules/.pnpm/@storybook+core-server@7.0.0-beta.13/node_modules/@storybook/core-server/dist/cjs/utils/StoryIndexGenerator.js:95:16)
>  ELIFECYCLE  Command failed with exit code 1.

`package.json`

```
{
	"type": "module",
	"scripts": {
		"storybook": "storybook dev -p 6006",
		"storybook:build": "storybook build"
	},
	"devDependencies": {
		"@storybook/addon-essentials": "7.0.0-beta.19",
		"@storybook/addon-interactions": "7.0.0-beta.19",
		"@storybook/addon-links": "7.0.0-beta.19",
		"@storybook/addon-svelte-csf": "^2.0.10",
		"@storybook/blocks": "7.0.0-alpha.8",
		"@storybook/svelte": "7.0.0-beta.19",
		"@storybook/sveltekit": "7.0.0-alpha.58",
		"@storybook/testing-library": "^0.0.13",
		"@sveltejs/kit": "^1.0.1",
		"storybook": "^7.0.0-beta.13",
		"svelte": "^3.54.0",
		"svelte-check": "^2.9.2",
		"typescript": "^4.9.4",
		"vite": "^4.0.4",
	}
}
```

`.storybook/main.cjs`

```
module.exports = {
	stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx|svelte)'],
	addons: [
		'@storybook/addon-links',
		'@storybook/addon-essentials',
		'@storybook/addon-interactions',
		'@storybook/addon-svelte-csf',
	],
	framework: {
		name: '@storybook/sveltekit',
		options: {},
	},
	docs: {
		docsPage: true,
	},
	features: {
		storyStoreV7: true,
	}
};
```

`Button.svelte`

```
<script lang="ts">
    export let text: string;
    export let primary: boolean = true;
</script>

<button>{text} {primary}</button>
```

`Button.stories.svelte`

```
<script lang="ts">
    export let text: string;
    export let primary: boolean = true;
</script>

<button>{text} {primary}</button>
```

**Comment by smart** [comment]
I'm having this issue too, did you find a resolution?

**Reply by smart** [reply]
I got an answer I had to use the next release - npm add @storybook/addon-svelte-csf@next
